src/test/java/org/broadinstitute/consent/http/service/DataAccessRequestServiceTest.java

```diff
@@ -228,6 +229,7 @@ void createProgressReport() {
     user.setLibraryCards(List.of(new LibraryCard()));
     parentDar.setUserId(user.getUserId());
     when(dataAccessRequestDAO.findByReferenceId(progressReport.getReferenceId())).thenReturn(progressReport);
+    when(dataAccessRequestDAO.findApprovedDatasetsByDar(parentDar.getReferenceId())).thenReturn(List.of(APPROVED_PROGRESS_REPORT_DATASET_ID));
```

Comment:
This code already has the dataset ID so there's no need for a named constant.
```suggestion
    when(dataAccessRequestDAO.findApprovedDatasetsByDar(parentDar.getReferenceId()))
        .thenReturn(progressReport.getDatasetIds());
```

(or `Set.copyOf(progressReport.getDatasetIds())` if you change the DAO method return type)

This value is from `generateDataAccessRequest()` here:
```
    dar.addDatasetId(1);
```